docker-compose.override.yml

```diff
@@ -73,27 +73,27 @@ services:
 ##        yarn link @centerforopenscience/ember-osf &&
 ##        yarn start --host 0.0.0.0 --port 4200 --live-reload-port 41953
 
-#  preprints:
-#    volumes:
-#      - preprints-sync:/code:nocopy
-#
-##      # Use this for ember-osf linked development (with docker-sync):
-##      - preprints_dist_vol:/code/dist
-##      - emberosf-sync:/ember-osf
-##    depends_on:
-##      - emberosf
-##    command:
-##      - /bin/bash
-##      - -c
-##      - cd /ember-osf &&
-##        yarn link &&
-##        cd /code &&
-##        (rm -r node_modules || true) &&
-##        yarn --frozen-lockfile &&
-##        yarn link @centerforopenscience/ember-osf &&
-##        (rm -r bower_components || true) &&
-##        ./node_modules/.bin/bower install --allow-root --config.interactive=false &&
-##        yarn start --host 0.0.0.0 --port 4201 --live-reload-port 41954
+  preprints:
```

Comment:
Those docker change doesn't seem to be related to the purpose of this pr. Are those necessary?

Reply:
No, I accidentally committed these.